.github/workflows/reusable_docker_pipeline.yml

```diff
@@ -240,4 +242,63 @@ jobs:
       
       - name: Inspect image
         run: |
-          docker buildx imagetools inspect ${{ vars.AWS_ECR_REGISTRY_ID }}/${{ needs.prepare-metadata.outputs.image-name }}:${{ steps.meta.outputs.version }}
\ No newline at end of file
+          docker buildx imagetools inspect ${{ vars.AWS_ECR_REGISTRY_ID }}/${{ needs.prepare-metadata.outputs.image-name }}:${{ steps.meta.outputs.version }}
+
+  vulnerability_scan:
+    runs-on: ubuntu-latest
+    # if: inputs.publish
```

Comment:
We may need the `if: inputs.publish` as the image needs to be published before it can be scanned?
https://github.com/babylonlabs-io/.github/blob/1cd090effda76cdcbdae0ab0b2e6226886a6f268/.github/workflows/reusable_docker_pipeline.yml#L117